### PR TITLE
LP-2619 Handle webinar reminders

### PR DESCRIPTION
**Pull request by asadali145**

### CODE REVIEW

**Review by zahid-arbisoft — APPROVED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1673/workflows/workflows_id/jobs/1673/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (70.6%): Missing lines 53,67,80,94,116
openedx/adg/common/lib/mandrill_client/helpers.py (50.0%): Missing lines 41-46,48-49
openedx/adg/common/lib/mandrill_client/tasks.py (70.0%): Missing lines 44,56-57
openedx/adg/lms/webinars/admin.py (90.9%): Missing lines 81
openedx/adg/lms/webinars/constants.py (100%)
openedx/adg/lms/webinars/helpers.py (98.0%): Missing lines 124
openedx/adg/lms/webinars/managers.py (100%)
openedx/adg/lms/webinars/migrations/0004_auto_20210424_1848.py (0.0%): Missing lines 3,6,8,12
openedx/adg/lms/webinars/models.py (93.9%): Missing lines 165,211
openedx/adg/lms/webinars/tasks.py (100%)
openedx/adg/lms/webinars/tests/conftest.py (100%)
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/tests/test_tasks.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   220 lines
Missing: 24 lines
Coverage: 89%
-------------
```

File: openedx/adg/common/lib/mandrill_client/tasks.py [hunk 1]
```diff
@@ -14,18 +14,21 @@
...
     Arguments:
         template (str): String containing template id
         emails (iterable): Email addresses of users
         context (dict): Dictionary containing email content
+        send_at (str): When this message should be sent as a UTC timestamp in YYYY-MM-DD HH:MM:SS format.
 
     Returns:
         None
     """
+    from openedx.adg.lms.webinars.helpers import save_scheduled_reminder_ids
```
Comment (emraan004) [inline openedx/adg/common/lib/mandrill_client/tasks.py:30]: I think mandril client is generic for all sort of emails but there is some webinars related code in this file. can we move it webinars app?

**Review by emraan004 — CHANGES_REQUESTED** [review]

File: openedx/adg/common/lib/mandrill_client/tasks.py (hunk 1 above)
Comment (asadali145) [reply openedx/adg/common/lib/mandrill_client/tasks.py:30]: Actually its the generic task that sends all the emails, So, in this case we want to store ids that mandrill returns. I have added the helper in webinars app but when mandrill returns the response we need to pass that response to the called method `save_scheduled_reminder_ids `.

Comment (asadali145) [reply openedx/adg/common/lib/mandrill_client/tasks.py:30]: Its like when we are sending reminder emails, just save the scheduled msg ids. All the save related logic is in webinars app.

Comment (emraan004) [reply openedx/adg/common/lib/mandrill_client/tasks.py:30]: Let's review this code in a team meeting and see if we need improve this code in a followup story 👍

**Review by emraan004 — APPROVED** [review]
👍

**Pull request merged**